docker-compose.yml

```diff
@@ -170,16 +170,17 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
-  nginx:
+  frontend:
     << : *restart_policy
     ports:
       - '9000:80/tcp'
-    image: "nginx:1.16"
-    volumes:
-      - type: bind
-        read_only: true
-        source: ./nginx
-        target: /etc/nginx
+    image: 'us.gcr.io/sentryio/frontend:latest'
```

Comment:
I am assuming you'd rather not pin a strict version here, but if you would, those will come soon and we can wait.

Reply:
On-premise users based in China cannot reach this registry so we need to put these images on Docker Hub if possible (somehow they are reachable inside).

Reply:
Also is this repo public yet?

Reply:
Sure, we can push it to docker hub at some point, and no, it's not public yet.